lib/cli_common/cli_common/command.py

```diff
@@ -82,8 +82,8 @@ def run_check(command, **kwargs):
             error=error,
         )
         raise click.ClickException(
-            'Command (`{}`) failed with code: {}.'.format(
-                command_as_string,
+            '`{}` failed with code: {}.'.format(
+                command[0],
```

Comment:
i know it is very unlikely but `command` could be an empty list.

Reply:
Should I just assert that it isn't an empty list?